<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="centralWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <widget class="QLabel" name="descriptionArticle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Description de l'article:</string>
        </property>
       </widget>
       <widget class="QLabel" name="prixArticle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix de l'article:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDescription">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editPrix">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxTax">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Article taxable</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouterBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="retirerBtn">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Retirer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reinitialiserBtn">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>150</y>
          <width>211</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Reinitialiser</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listArticles">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>211</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelArticle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Articles</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTotAvantTaxes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total avant taxes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTotaltaxes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total des taxes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTotal">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalAvantTaxes">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00$</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalTaxes">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>330</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00$</string>
        </property>
       </widget>
       <widget class="QLabel" name="total">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>350</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00$</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPrix">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>190</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
